.gitignore

```diff
@@ -77,6 +78,7 @@ tests/__init__.py
 
 # Development task artifacts
 default.db
+csv/
 
 # virtualenvs
-bulk-grades-env/
\ No newline at end of file
+bulk-grades-env/
```

Comment:
this isn't a blocker but I believe the (unofficial?) standard is to make the venv the same name as the repo